definitions/divorce/json/CaseEvent.json

```diff
@@ -2422,5 +2422,17 @@
     "PostConditionState": "*",
     "SecurityClassification": "PUBLIC",
     "ShowSummary": "Yes"
+  },
+  {
+    "LiveFrom": "23/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "solApplyForDA",
+    "Name": "Solicitior applies for DA",
```

Comment:
Will also need authorisation for this event -
[PETSOLICITOR] - CRU
caseworker-divorce-courtadmin_beta - R
caseworker-divorce-courtadmin - R
caseworker-divorce-superuser - R
caseworker-divorce-courtadmin-la - R